### PR TITLE
Clarify multi-node transaction isolation levels

### PR DESCRIPTION
**Pull request by mfundul**
# Description

Clarify multi-node transaction isolation levels.

# Version

Which documentation version does this PR apply to?

- [X] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

### CODE REVIEW

File: timescaledb/how-to-guides/multi-node-setup/required-configuration.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ data nodes (if not already set, `150` is recommended).
   through the access node configuration if desired. This setting is disabled
   by default in PostgreSQL, but may be worth verifying in your specific
   environment.
+* If the transaction isolation level is set to `READ COMMITTED` it is
+  automatically upgraded to `REPEATABLE READ` whenever a distributed operation
+  takes place for consistency reasons.
```
Comment (mkindahl) [inline timescaledb/how-to-guides/multi-node-setup/required-configuration.md:21]: Mostly for clarity.
```suggestion
* If the transaction isolation level is set to `READ COMMITTED` it is
  automatically upgraded to `REPEATABLE READ` whenever a distributed operation
  takes place for consistency reasons. If the isolation level is `READ COMMITTED` or `SERIALIZABLE`, it is not changed.
```

**Review by mkindahl — COMMENTED** [review]
LGTM, suggest a minor clarification for completeness.

File: timescaledb/how-to-guides/multi-node-setup/required-configuration.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ data nodes (if not already set, `150` is recommended).
   through the access node configuration if desired. This setting is disabled
   by default in PostgreSQL, but may be worth verifying in your specific
   environment.
+* If the transaction isolation level is set to `READ COMMITTED` it is
+  automatically upgraded to `REPEATABLE READ` whenever a distributed operation
+  takes place for consistency reasons. If the isolation level is `READ COMMITTED` or `SERIALIZABLE`, it is not changed.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/multi-node-setup/required-configuration.md:21]:
```suggestion
* For consistency, if the transaction isolation level is set to `READ COMMITTED` it is
  automatically upgraded to `REPEATABLE READ` whenever a distributed operation
  takes place. If the isolation level is `READ COMMITTED` or `SERIALIZABLE`, it is not changed.
```

**Review by Loquacity — APPROVED** [review]
Minor edit for wording, non-blocking.

**Review by ryanbooz — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

**Pull request merged**